draft-ietf-httpbis-message-signatures.md

```diff
@@ -778,23 +802,25 @@ In order to create a signature, a signer MUST follow the following algorithm:
 
 3. If applicable, the signer sets the signature's expiration time property to the time at which the signature is to expire.
 
-4. The signer creates an ordered list of content identifiers representing the message content and signature metadata to be covered by the signature, and assigns this list as the signature's Covered Content.
-   * Once an order of covered content is chosen, the order MUST NOT change for the life of the signature.
-   * Each covered content identifier MUST either reference an HTTP header in the request message {{http-header}} or reference a specialty content field listed in {{specialty-content}} or its associated registry.
-   * Signers of a request SHOULD include some or all of the control data in the covered content list, such as the `@method`, `@authority`, `@target-uri`, or some combination thereof.
+4. The signer creates the signature parameters, which is an ordered set of component identifiers representing the message components to be covered by the signature, and assigns signature metadata parameters to this value. The value of this is later used as the value of the `Signature-Input` field as described in {{signature-input-header}}.
```

Comment:
```
4. The signer creates the signature parameters, which is ...
```
is the wording correct? Is "signature parameters" a definition?

Reply:
Yes, it's described here in this sentence.

Reply:
Updated language here to be more clear as to what's being created by the signer -- it's a data structure consisting of an order set with a bunch of parameters.